db/migrate/20200329100626_create_notes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notes do |t|
+      t.text :note
```

Comment:
I'd add `null: fase`.

Reply:
I think a name like `body` or `content` or something similar, would be more suitable, as otherwise we'd have `note.note`.

Reply:
I was going from the database design

Reply:
Yeah, sorry, I agree.

Reply:
I've updated as per your suggestions @AlexVPopov